package.json

```diff
@@ -4,6 +4,7 @@
   "license": "MIT",
   "scripts": {
     "start": "ng serve",
+    "start:https": "ng serve --ssl true --host 0.0.0.0 --port 8443",
```

Comment:
Also we should probably mention how / where to make changes to make OAuth using popup instead of page refresh in this README.md section